liferay-lima/liferay-portal
# LPS-200907 Video streaming subtitles

Now subtitles are visible if they are provided via config to the Fragment. 
This is done in a new module without test, so no other test must be affected. If ci:test:relevant doesn't pass I will force forward. 

**Steps to test it:**

1. Edit home page and add **Video Streaming** fragment
2. Configure it. **URL** https://s3-us-west-2.amazonaws.com/s.cdpn.io/122116/2010-0927-KMOV-News-4.mp4 and **subtitles** https://s3-us-west-2.amazonaws.com/s.cdpn.io/122116/2010-0518-KMOV-News-4_3.vtt 
3. Publish page. To see subtitles, you must select it from the control bar. 

<img width="939" alt="Screenshot 2023-11-15 at 13 15 03" src="https://github.com/liferay-lima/liferay-portal/assets/8373764/852130e3-fc88-4301-a09a-0becc0a2780f">
